components/safe_browsing/android/java/src/org/chromium/components/safe_browsing/BraveSafeBrowsingUtils.java

```diff
@@ -0,0 +1,143 @@
+        int POTENTIALLY_HARMFUL_APPLICATION = 4;
+        int SOCIAL_ENGINEERING = 5;
+        int SUBRESOURCE_FILTER = 13;
+        int BILLING = 15;
+        int CSD_ALLOWLIST = 16;
+        int MAX_VALUE = 17;
+    }
+
+    @IntDef({
+        SafeBrowsingJavaResponseStatus.SUCCESS_WITH_LOCAL_BLOCKLIST,
+        SafeBrowsingJavaResponseStatus.SUCCESS_WITH_REAL_TIME,
+        SafeBrowsingJavaResponseStatus.SUCCESS_FALLBACK_REAL_TIME_TIMEOUT,
+        SafeBrowsingJavaResponseStatus.SUCCESS_FALLBACK_REAL_TIME_THROTTLED,
+        SafeBrowsingJavaResponseStatus.FAILURE_NETWORK_UNAVAILABLE,
+        SafeBrowsingJavaResponseStatus.FAILURE_BLOCK_LIST_UNAVAILABLE,
+        SafeBrowsingJavaResponseStatus.FAILURE_INVALID_URL
+    })
+    public @interface SafeBrowsingJavaResponseStatus {
+        int SUCCESS_WITH_LOCAL_BLOCKLIST = 0;
+        int SUCCESS_WITH_REAL_TIME = 1;
+        int SUCCESS_FALLBACK_REAL_TIME_TIMEOUT = 2;
+        int SUCCESS_FALLBACK_REAL_TIME_THROTTLED = 3;
+        int FAILURE_NETWORK_UNAVAILABLE = 4;
+        int FAILURE_BLOCK_LIST_UNAVAILABLE = 5;
+        int FAILURE_INVALID_URL = 6;
+    };
+
+    @IntDef({
+        SafeBrowsingJavaThreatType.NO_THREAT,
+        SafeBrowsingJavaThreatType.SOCIAL_ENGINEERING,
+        SafeBrowsingJavaThreatType.UNWANTED_SOFTWARE,
+        SafeBrowsingJavaThreatType.POTENTIALLY_HARMFUL_APPLICATION,
+        SafeBrowsingJavaThreatType.BILLING,
+        SafeBrowsingJavaThreatType.ABUSIVE_EXPERIENCE_VIOLATION,
+        SafeBrowsingJavaThreatType.BETTER_ADS_VIOLATION
+    })
+    public @interface SafeBrowsingJavaThreatType {
+        int NO_THREAT = 0;
+        int SOCIAL_ENGINEERING = 2;
+        int UNWANTED_SOFTWARE = 3;
+        int POTENTIALLY_HARMFUL_APPLICATION = 4;
+        int BILLING = 15;
+        int ABUSIVE_EXPERIENCE_VIOLATION = 20;
+        int BETTER_ADS_VIOLATION = 21;
+    };
+
+    private static int safeBrowsingToSafetyNetJavaThreatType(int safeBrowsingThreatType) {
+        switch (safeBrowsingThreatType) {
+            case SafeBrowsingJavaThreatType.NO_THREAT:
+                return 0;
+            case SafeBrowsingJavaThreatType.SOCIAL_ENGINEERING:
+                return SafetyNetJavaThreatType.SOCIAL_ENGINEERING;
+            case SafeBrowsingJavaThreatType.UNWANTED_SOFTWARE:
+                return SafetyNetJavaThreatType.UNWANTED_SOFTWARE;
+            case SafeBrowsingJavaThreatType.POTENTIALLY_HARMFUL_APPLICATION:
+                return SafetyNetJavaThreatType.POTENTIALLY_HARMFUL_APPLICATION;
+            case SafeBrowsingJavaThreatType.BILLING:
+                return SafetyNetJavaThreatType.BILLING;
+            case SafeBrowsingJavaThreatType.ABUSIVE_EXPERIENCE_VIOLATION:
+            case SafeBrowsingJavaThreatType.BETTER_ADS_VIOLATION:
+                // See SafeBrowsingJavaToSBThreatType at safe_browsing_api_handler_bridge.cc
+                return SafetyNetJavaThreatType.SUBRESOURCE_FILTER;
+            default:
+                Log.w(
```

Comment:
print in debug only?

Reply:
yes, thanks, https://github.com/brave/brave-core/pull/25842/commits/dd4c302cc51dec3f3409b0d5667d8ce055459b5c#diff-fbe6ef23e5c73345c352043cf4cf6e37296db1fd0b59b63036bfd029124187b8R212